plugins/inputs/statsd/statsd.go

```diff
@@ -261,41 +287,43 @@ func (s *Statsd) Gather(acc telegraf.Accumulator) error {
 			}
 		}
 
-		acc.AddFields(metric.name, fields, metric.tags, now)
+		acc.AddFields(m.name, fields, m.tags, now)
 	}
 	if s.DeleteTimings {
 		s.timings = make(map[string]cachedtimings)
 	}
 
-	for _, metric := range s.gauges {
-		acc.AddGauge(metric.name, metric.fields, metric.tags, now)
+	for _, m := range s.gauges {
+		acc.AddGauge(m.name, m.fields, m.tags, now)
```

Comment:
This is fine but normally don't change this type of stuff, it makes review and tracing history more difficult.

Reply:
It had collision with a type.

Reply:
Okay but in the future don't worry about it.